src/Illuminate/Contracts/Database/Eloquent/Hydrator.php

```diff
@@ -2,13 +2,17 @@
 
 namespace Illuminate\Contracts\Database\Eloquent;
 
+use Illuminate\Database\Eloquent\Model;
+
 interface Hydrator
 {
     /**
      * Fill a new Eloquent model instance with raw attributes returned from the query builder.
      *
+     * @param Model $model
```

Comment:
Please use fully qualified class names in phpdoc, and put two spaces either side of the type within param docs.